bin/debian-mirror

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+help() {
+  echo "Usage: $0 [OPTIONS]" >&2
+  echo "" >&2
+  echo "Create a local debian mirror for ffhi-prepare" >&2
+  echo "" >&2
+  echo "Options:" >&2
+  echo "" >&2
+  echo "-m, --mirror-path PATH        specify path for debian mirror" >&2
+  echo "                              default: \$HOME/debian-mirror" >&2
+  echo "-d, --dist DIST1[,DIST2,...]  debian distros to mirror" >&2
+  echo "                              default: all dists referenced in ansible inventory" >&2
+  exit 1
+}
+
+#
+# parse options
+#
+
+OPTS=$(getopt -o m:d:h --long mirror-path:,dist:,help -n 'parse-options' -- "$@")
+[ "$?" != "0" ] && help
+
+eval set -- "$OPTS"
+
+OPT_MIRROR_PATH=${HOME}/debian-mirror
+OPT_DISTS=$(ansible-inventory --list | grep ansible_host_dist | sed -e "s/.*\://g;s/[[:space:]]//g;s/[\",]//g" | sort -u)
```

Comment:
I think OPT_DISTS will result in "stretch testing ....". debmirror takes a comma separated list for --dist option